bin/manifest.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import base64, os, random, sys
+from ripple import ed25519
+from ripple import ecdsa
+from hashlib import sha256
+
+#-----------------------------------------------------------
+
+# 58 character alphabet used for Ripple
+human_alphabet = 'rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz'
+
+def base_encode(b, digits, check = 1):
+    """ Returns binary as a radix-encoded string """
+    lb = b.lstrip(b'\0')
+    bn = long(0)
+    for c in lb:
+        bn = bn * 256 + ord(c)
+    radix = len(digits)
+    s = ''
+    while (bn > 0):
+        bn, mod = divmod(bn, radix)
+        s += digits[mod]
+    s = digits[0] * (len(b) - len(lb)) + s[::-1]
+    if (check):
+        assert base_decode(s, digits, 0) == b
+    return s
+
+def base_decode(s, digits, check = 1):
+    """ Returns the binary from decoding a radix-encoded string """
+    bn = 0
+    radix = len(digits)
+    ls = s.lstrip(str(digits[0]))
+    for c in ls:
+        bn = bn * radix + digits.index(c)
+    b = ''
+    while bn > 0:
+        bn, m = divmod(bn, 256)
+        b += chr(m)
+    b = b'\0' * (len(s) - len(ls)) + b[::-1]
+    if (check):
+        assert base_encode(b, digits, 0) == s
+    return b
+
+def base58encode(b):
```

Comment:
Do you ever use the two functions above?  why not just make the default for `digits` to be `human_alphabet` in the above function?